Getting Started Guide: add instructions for ITAC, update chatqna steps (#303)
opea-project/docs: getting-started/README.md

10. Click Go to resource -> Connect -> Connect -> SSH using Azure CLI. Accept the terms and then select "Configure + connect"

> >**Note**: If you have issues connecting to the instance with SSH, you could use instead Bastion with your username and password.

Reviewer: ```suggestion
>**Note**: If you have issues connecting to the instance with SSH, you could instead access the same via the Bastion host with your username and password.

```

Author: applied

9. Note the public IP address of the machine once its launched. 

10. Once the instance is launched, click on the subnet in the Primary VNIC section. Then click on the "Default Security List for vcn-xxxxxxxx-xxxx" , click on the "Add Ingress Rules".  Add the following information:
    - Source CIDR: **0.0.0.0/0**  
    - Source Port Range: **All** 
    - Destination Port Range: **80** 
    - Click on "Save"

11. Connect using ssh (`ssh -i <private_key>  ubuntu@<public_ip_address>`).

:::
:::{tab-item} Intel® Tiber™ AI Cloud
:sync: ITAC

> 1. Log in to [Intel® Tiber™ AI Cloud](https://ai.cloud.intel.com/) - Go to the "Compute" tab on the left and click on "Instances". In the center of the screen, click on the "Launch instance" button.

Reviewer: Do they already have an account/password set up? Might they already have cloud credits. Do we need to point them to some link?

Perhaps just start off with "We assume you already have an Intel Tiber AI Cloud account".

Author: added some instructions around creating an account and getting cloud credits

2. Select your instance configuration, instance type, and machine image which will be Ubuntu.

>**Note**: It is recommended to use the `VM-SPR-LRG` powered by 4th Generation Intel® Xeon® Scalable processors with 64GB of memory and 64GB of disk or more if you wish to use a CPU to run an 8B-parameter model.

3. Fill out the rest of the form such as giving your instance a name and answering any additional quesitons.

> 4. Add your public key for SSH. You can select a key you have previously uploaded or upload your own key. The "Upload Key" button will also contain instructions to create a new SSH key.

Reviewer: ```suggestion
4. Add your public key for SSH. You can select a key you have previously uploaded or upload a key. The "Upload Key" button also provides instructions on how to create a new SSH key.

```

Author: applied

5. Click "Launch instance" to start your machine.

6. Go back to the "Compute" tab and under "Instances", note down the private IP address of your new VM.

7. If you wish to make the UI accessible to others, proceed to the next step to create a load balancer. Otherwise, skip to Step 10 which will explain how to connect to your VM with port forwarding.

8. Create a load balancer. This can be found in Compute->Load Balancers. Click on "Launch Load Balancer". Ignore any messages about signing up for access and close any pop-up windows if any. Fill out the form with the following info: 
   - Name: **Name for your load balancer**
   - Source IP: **The private IP address of your VM in Step 6**
   - Listener Port: **80**
   - Instance Port: **80**
   - Monitor Type: **HTTP**
   - Mode: **Round Robin**

>    - Instances: **The VM you created**

Reviewer: unclear what needs to be entered here

Author: add more details

   >**Note**: The port used is 80 because this is the NGINX port for the GenAI Examples.

>    Click "Launch" when ready.

Reviewer: what has to be ready? Assuming VM instance ready because you have IP.

Author: Ready means when the user is finished with the load balancer settings. For simplicity, I removed it.

```

Set the required environment variables:
```bash
export host_ip="localhost"
export HUGGINGFACEHUB_API_TOKEN="Your_Huggingface_API_Token"
```

Set up proxies if you are behind a firewall:
```bash
export no_proxy=${your_no_proxy},$host_ip
export http_proxy=${your_http_proxy}
export https_proxy=${your_http_proxy}
```

Set up other specific use-case environment variables in `set_env.sh` before running it. For example, this is where you can change the model(s) to run with.
```bash
cd ChatQnA/docker_compose/intel/cpu/xeon/
source set_env.sh
```

Now we can start the services:
```bash
docker compose -f compose.yaml up -d
```
>**Note**: It takes a few minutes for the services to start. Check the logs for the services to ensure that ChatQnA is running before proceeding further.

For example to check the logs for the `vllm-service`:

```bash
docker logs vllm-service | grep Complete
```
Proceed further **only after** the output shows `Application startup complete.` as shown:
```bash
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:80 (Press CTRL+C to quit)
```

> Run `docker ps -a` as an additional check to verify that all the services are running as shown:

Reviewer: Add a sentence like .. "Note the 1.2 in the image name matches the release version that was specified in the git checkout command after cloning. "

Author: added
